externals/autotools/share/autoconf/autom4te.cfg

```diff
@@ -106,7 +106,7 @@ end-language: "Autoreconf-preselections"
 # This intermediate language is used by aclocal to build aclocal.m4.
 
 begin-language: "Autoconf-without-aclocal-m4"
-args: --prepend-include '/Users/economon/SU2/externals/autotools/share/autoconf'
```

Comment:
Maybe you can revert those changes in the autotools directory.

Reply:
Do you mean changing all those home directories back to `economon`? I think all of this happened automatically when I changed the config to look for the proper Python dirs.

Reply:
I am referring to all of the changes in the externals/autotools folder.